daemon/container_unix.go

```diff
@@ -640,7 +640,7 @@ func (container *Container) updateNetworkSettings(n libnetwork.Network, ep libne
 		return err
 	}
 
-	if container.hostConfig.NetworkMode == runconfig.NetworkMode("bridge") {
+	if container.hostConfig.NetworkMode == runconfig.NetworkMode("default") {
```

Comment:
@mavenugo Although n.Type() or  NetworkMode.IsBridge() can tell if this is a bridge driver, they do not provide the bridge name. I do not think libnetwork currently has a method to expose the name. This cause "docker inspect" displace an empty bridge. Do you think adding a new method to libnetwork.network interface will help? Thanks.